modules/olmv1-red-hat-catalogs.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_content-type: REFERENCE
+
+[id="olmv1-red-hat-catalogs_{context}"]
+= Red Hat-provided Operator catalogs in OLM 1.0
+
+Operator Lifecycle Manager (OLM) 1.0 does not include Red Hat-provided Operator catalogs by default. If you want to add a Red Hat-provided catalog to your cluster, create a custom resource (CR) for the catalog and apply it to the cluster. The following custom resource (CR) examples show how to create a catalog resources for OLM 1.0.
+
```

Comment:
@Xia-Zhao-rh, in the ticket, https://issues.redhat.com//browse/OSDOCS-6689, only these three were specified. I wasn't sure if this was because Marketplace subscriptions aren't supported yet. I will do some research on this today.

Reply:
@kevinrizza or @tlwu2013 Are you able to clarify this issue? Should the Marketplace catalog be intentionally left out of the catalog examples? Or was that an oversight in the ticket?

Reply:
The redhat-marketplace catalog is built by IBM. I believe it is converted to FBC so it should "work" here, but since redhat has less active control over it as the schema evolves I imagine it would be better to leave it out of "try it out" documentation because changes or fixes if things don't work with v1 will take longer to resolve. We definitely don't test this right now with v1, anyway.